src/main/java/customerrelationsmanagement/Tables.java

```diff
@@ -140,9 +140,14 @@ public String creationString() {
 				 "INT(6)"
 				};
 			}
-			// Todo Adam return the string array of mysql types 
-			//  for each column in your new table instead of null
-			case SUPPLIER -> {return null;}
+			case SUPPLIER -> {
```

Comment:
Glad to welcome supplier to our new enum class. Let's make sure nothing gets mixed up with the addition of a new table. But since the tables class is now an enum, hopefully that makes it easier to take care of and maintain due to it's consistency.